beacon/endpoints/rest/schemas/alternative.py

```diff
@@ -158,7 +160,7 @@ def ga4gh_phenopackets_variant_annotation_v10(row):
             'hgvs': hgvs_id
         },
         'zygosity': None,
-    } for hgvs_id in row['transcript_hgvs_ids'] or []]
+    } for hgvs_id in row['transcript_hgvs_ids']]
```

Comment:
You made sure the `row['transcript_hgvs_ids']` is always a list? (actually, an _iterable_), and that it can't be None?

Reply:
Yes, I changed the function so it always returns empty array.

Reply:
Maybe just add a comment then?